scripts/badger/aura_tcl_strategy.py

```diff
@@ -0,0 +1,136 @@
+
+    # 2.1 swap rewards for usdc
+    vault.cow.market_sell(bal, usdc, bal_swap_for_usdc, deadline=60 * 60 * 4, coef=COEF)
+    vault.cow.market_sell(
+        aura, usdc, aura_swap_for_usdc, deadline=60 * 60 * 4, coef=COEF
+    )
+
+    # 2.2 send to voter and deposit into aurabal/bauraBAL
+    aura.transfer(voter, balance_aura - aura_swap_for_usdc)
+
+    bal_to_deposit = balance_bal - bal_swap_for_usdc
+
+    # wrapper min estimation
+    wrapper_aurabal_out = wrapper.getMinOut(bal_to_deposit, 9950)
+
+    # bpt out and swap estimation
+    bpt_out = vault.balancer.get_amount_bpt_out(
+        [bal, weth], [bal_to_deposit, 0], pool=b80bal_20weth
+    ) * Decimal(SLIPPAGE)
+    amt_pool_swapped_out = float(vault.balancer.get_amount_out(
+        b80bal_20weth, auraBAL, bpt_out, pool=bauraBAL_stable
+    ) * Decimal(SLIPPAGE))
+
+    if amt_pool_swapped_out > wrapper_aurabal_out:
+        console.print(
+            "[green] === INFO: better outcome of auraBAL via depositing and swapping === [/green]"
+        )
+        console.print(
+            f"[green] === Extra {(amt_pool_swapped_out-wrapper_aurabal_out)/1e18} auraBAL received === [/green]"
+        )
+        vault.balancer.deposit_and_stake(
+            [bal, weth], [bal_to_deposit, 0], pool=b80bal_20weth, stake=False
+        )
+        balance_bpt = b80bal_20weth.balanceOf(vault) * SLIPPAGE
+        vault.balancer.swap(b80bal_20weth, auraBAL, balance_bpt, pool=bauraBAL_stable)
+    else:
+        console.print(
+            "[green] === INFO: better outcome of auraBAL via wrapper === [/green]"
+        )
+        console.print(
+            f"[green] === Extra {(wrapper_aurabal_out - amt_pool_swapped_out)/1e18} auraBAL received === [/green]"
+        )
+        bal.approve(wrapper, bal_to_deposit)
+        wrapper.deposit(
+            bal_to_deposit,  # uint256 _amount
+            wrapper_aurabal_out,  # uint256 _minOut
+            False,  # bool _lock
+            ZERO_ADDRESS,  # address _stakeAddress
+        )
+
+    # 3. dogfood in bauraBAL if in prod
+    vault_status = registry_v_2.productionVaultInfoByVault(bauraBal)[2]
+    # check if vault is `open`, then dogfood
+    if vault_status == 3:
+        auraBAL.approve(bauraBal, 2 ** 256 - 1)
+        bauraBal.depositAll()
+        auraBAL.approve(bauraBal, 0)
+
+    # 3. tree claim
+    if bauraBal.balanceOf(vault) > 0:
+        vault.init_badger()
+        # will claim: bcvxCRV, bveCVX & graviAURA
+        # NOTE: should we just narrow the claim for graviAURA and modify `claim_all`?
+        # NOTE: short <2h window for signing as new cycles may be posted
+        vault.badger.claim_all()
+
+        # 3.1: send to tree for deficit coverage over time.
+        # will help together with the post-thresher trasnfer
+        bcvxcrv.transfer(r.badger_wallets.badgertree, bcvxcrv.balanceOf(vault))
+
+        # 3.2: send influence asset to voter
+        bvecvx.trasnfer(voter, bvecvx.balanceOf(vault))
```

Comment:
```suggestion
        bvecvx.transfer(voter, bvecvx.balanceOf(vault))
```

Reply:
removed this section, makes sense specially for the sensitivity of getting it sign under a narrow time window